h2o-py/h2o/model/model_base.py

```diff
@@ -62,6 +62,40 @@ def params(self):
                          "actual": self.parms[p]["actual_value"]}
         return params
 
+    @property
+    def default_params(self):
+        """
+        Get default parameters of a model
+
+        :return: A dictionary of default parameters for the model
+        """
+        params = {}
+        for p in self.parms:
+            params[p] = self.parms[p]["default_value"]
+        return params
+
+    @property
+    def actual_params(self):
+        """
+        Get actual parameters of a model
+
+        :return: A dictionary of actual parameters for the model
+        """
+        params_to_select = {'model_id':'name', \
+                            'response_column':'column_name', \
+                            'training_frame': 'name', \
+                            'validation_frame':'name'}
+        params = {}
+        for p in self.parms:
+            if p in params_to_select.keys():
+                try:
+                    params[p] = self.parms[p]["actual_value"][params_to_select[p]]
+                except:
+                    params[p] = None
```

Comment:
So you want to write `params[p] = self.parms[p]["actual_value"].get(params_to_select[p], None)` then? Exceptions in Python (and in Java) should only be used for exceptional situations, not for regular execution flow...